packages/pasteup/typography.ts

```diff
@@ -41,14 +43,42 @@ const fontScaleMapping: any = {
     },
 };
 
-export const fontSize = (typex: string, level: number) =>
-    fontScaleMapping[typex][level].fontSize.toString();
+const fontSizeNumber = (category: Category, level: number): number =>
+    fontScaleMapping[category][level].fontSize;
+
+const lineHeightNumber = (category: Category, level: number): number =>
+    fontScaleMapping[category][level].lineHeight;
+
+const fontSizeCss = (category: Category, level: number): string =>
+    `font-size: ${fontSizeNumber(category, level)}px`;
+
+const lineHeightCss = (category: Category, level: number): string =>
+    `line-height: ${lineHeightNumber(category, level)}px`;
+
+const fontFamily = (category: Category): string => {
+    switch (category) {
+        case 'headline': {
+            return serif.headline;
+            break;
+        }
+        case 'body': {
+            return serif.body;
+            break;
+        }
+        default: {
+            return sans.body;
+            break;
+        }
+    }
```

Comment:
This could be a dictionary rather than a switch. Something like:

```
const families: { [Category]: string } = {
  headline: serif.headline,
  body: serif.body,
  textSan: sans.body
}

return families[category]
```
 would be more type-safe, and doesn't require an arbitrary `default` case.

Reply:
Thanks @SiAdcock, TIL, but the syntax is slightly more unfriendly:

```
    const families: { [category in Category]: string } = {
      headline: serif.headline,
      body: serif.body,
      textSans: sans.body
    }
    return families[category];
```